docs/scenarios/parametric_contributing.md

```diff
@@ -0,0 +1,60 @@
+# Contributing to Parametric System-tests
+
+Note: a more in-depth overview of parametric system-tests can be found in [parametric.md](parametric.md).
+
+**MUST:** Acquaint yourself with [this section](parametric.md#architecture-how-system-tests-work) for reference so you understand/can track what system-tests are actually doing.
+
+## Use cases
+
+Let's figure out if your feature is a good candidate to be tested with parametric system-tests. Parametric system-tests are great for assuring uniform behavior between tracers e.g. [environment variable configuration effects on api methods, sampling, propagation, configuration, telemetry](/tests/parametric).
```

Comment:
I think it's quite important to emphase here the fact that "parametric" stands for parameters. The original purpose of parametric scenario us when a behavior must be tested across several different value of a parameter. 

Using end-to-end scenarios for this use case is a pain, as we need to spawn an infra for each parameter value. But parametric scenario is way better, as it starts on container per test case, and it's reasonnably fast.

But if your usage does not implies several parameters, then an end-to-end scenario **must** be used : it'll achieve the same level of behavior uniformity verification. And it'll test the feature on real world use case, catching way more issues.

I think making this point very clear is very important, as parametric scenario has been a little bit abused by the past. And is it does not scale very well, if we can keep it for the relevant use case, it'll be for the best of all users.

Reply:
Thanks for this explanation! I'll add pieces of it in!